### DISCUSSION TITLE
Touch not working on ILI9488 driver based 3.5" TFT Touchscreen- Wemos Lolin32 MCU

### DISCUSSION THREAD

**Discussion by Surendra-Gayke** [Bodmer/TFT_eSPI#1468] [Q&A - Touch]
I am trying to interface 3.5" SPI TFT touchscreen to Wemos lolin32 (Esp wroom 32 based ) Microcontroller.
My Display program worked fine, But Touch program is not working.
Basically touchscreen is not responding to touch.

I am using following configurations is User_Setup.h file.

#define ILI9488_DRIVER

#define TFT_MISO 19 // it is not connected to TFT T_DO
#define TFT_MOSI 23
#define TFT_SCLK 18
#define TFT_CS 5 // Chip select control pin
#define TFT_DC 2 // Data Command control pin
#define TFT_RST 16 // Reset pin (could connect to RST pin)
#define TOUCH_CS 15
[User_Setup.txt](https://github.com/Bodmer/TFT_eSPI/files/7653739/User_Setup.txt)

**Comment by Bodmer** [comment]
Solved by user in #1466
